src/16F740.c

```diff
@@ -116,15 +120,15 @@ void btl_state_update_switch_to_player(void) {
     s32 i;
 
     if (gBattleState2 == 0) {
-        gBattleStatus.flags1 &= ~0x80000;
+        gBattleStatus.flags1 &= ~BS_FLAGS1_80000;
         reset_actor_turn_info();
-        gBattleStatus.selectedMoveID = 0;
+        gBattleStatus.selectedMoveID = MOVE_NOTHING;
```

Comment:
I think this should be MOVE_NONE (thoughts @nanaian?) - I realize you didn't add it in this PR though

Reply:
I'm ok with that